package.json

```diff
@@ -58,6 +58,7 @@
   },
   "scripts": {
     "ft_test": "mocha tester.js --compilers js:babel-core/register",
+    "test_aws": "babel-node tests/functional/aws-node-sdk/test/legacy/test.js",
```

Comment:
I assume we don't want this in the final PR.  Note that this is not all of the node aws sdk tests.